cassandra/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openzipkin/zipkin-base:1.4
+FROM quay.io/openzipkin/zipkin-base:base-0.0.0
```

Comment:
The tag (`base-0.0.0`) will be overwritten by the first release, and there's currently no meaningful value for it. This is just a placeholder.